packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantBadges/RestaurantBadge.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <span
+        :class="[
+            [$style['c-restaurantBadge']], {
+                [$style['.c-restaurantBadge--isLarge']]: isLarge
+            }]"
+        :style="`color:${textColour}; background-color:${backgroundColour};`"
+        :title="text"
+        data-test-id="restaurant-badge">
+        {{ text }}
+    </span>
+</template>
+
+<script>
+export default {
+    name: 'RestaurantBadge',
+    props: {
+        text: {
+            type: String,
+            required: true
+        },
+        textColour: {
+            type: String,
+            default: '#242e30'
```

Comment:
I don't have a good solution in mind yet, but I don't really like how it's hardcoded. If fozzie tokens change - we'll need to manually change all such values, and it's likely that we'll forget to do it.

Reply:
I agree - I can't see another way at the moment but open to ideas

Reply:
my initial solution hard coded some css modifier classes, so a consumer could pass in a colour scheme token like 'warm' or 'dark' with pie design tokens being leveraged. But then we have to add N classes for N colour and background colour combos. So I'm not sure which is worse.

Reply:
yeah, may be fozzie variables on the project level. Don't know..

Reply:
@ashleynolan do you have any thoughts on how I could handle this? I think it's reasonable to assume we will get X badge colour requests in the future given we currently already need to support this kind of change. We could always just add another modifier class each time (could end up not being many) but it feels pretty inflexible

Reply:
yeah if we could define a set of global JS vars that could work :P

Reply:
Does:
```js
import PieTokens from `@justeat/pie-design-tokens/dist/tokens.json;
const textColour = PieTokens.theme.jet.color.alias.default['content-default'];
```
not work, for example?